lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -82,13 +91,13 @@ def verify_response_block_list(self, response):
             self.non_orphaned_block_usage_keys,
         )
 
-    def verify_response_block_dict(self, response):
+    def verify_response_block_dict(self, response, block_usage_key=None):
```

Comment:
Is the `block_usage_key` optional parameter used?  I don't see a caller passing it in.